### ISSUE TITLE
Indicator Not Show Correctly When selectedPage changed

### ISSUE THREAD

**Issue by kentplaza** [manijak/nativescript-carousel#14]
I trying to make the carousel to be loop cycle (mean go to first when reach end or vice versa). Does it support?

For the mean time, I didn't see any function to allow feature above so that I try to write my own code  "pageChanged" event as below:

var carousel = page.getViewById("carousel");
carousel.selectedPage = 4;

But I found the indicator not going to show to correct selected item.

**Comment by manijak** [comment]
What page does it select? Keep in mind that `selectedPage` is an index. So the first one is 0. If you have 5 pages, the index of the last one would be 4. 

The infinite loop is only available on iOS, not Android for the time.

**Comment by kentplaza** [comment]
Yes. So that;s why i doing on my own code. If reach to last page, I will set carousel.selectedPage = 0.
The page will go to first page but the indicator will showing at last position.

**Comment by manijak** [comment]
Hmm.. I don't think you'll be able to do this outside the plugin. It is currently not possible to change the selected indicator while the pageChange event is running.

I'll have to take a look at this and add it as a future feature.
